src/freenet/node/MasterKeys.java

```diff
@@ -346,8 +336,13 @@ public static void killMasterKeys(File masterKeysFile) throws IOException {
 		FileUtil.secureDelete(masterKeysFile);
 	}
 
-	public DatabaseKey createDatabaseKey(Random random) {
-	    return new DatabaseKey(databaseKey, random);
+	@Deprecated
```

Comment:
Can you make this method call the other `createDatabaseKey()` method and also put that in a comment so that people using this know what they should be using instead of this method?

Reply:
Improved this a bit and added some Javadoc comments.